### ISSUE TITLE
Consider sticking to the original signers and using Google Forms for extra signatures

### ISSUE THREAD

**Issue by domenic** [dear-github/dear-github#19]
Compare to https://extensiblewebmanifesto.org/. As we are already seeing merge conflicts abound, people aren't following the single-project-per-person format, and aren't alphabetizing.

**Comment by sindresorhus** [comment]
:+1: Manually merging PRs for every signature is a lot of work and will result in lots of merge conflicts.

**Comment by thejameskyle** [comment]
heh, yeah I just caught up enough to look at this issue. I'm down

**Comment by simonwelsh** [comment]
`README.md merge=union` in `.gitattributes` should fix it, assuming Github's merge button obeys attributes.

**Comment by kangax** [comment]
Sounds good to me.

**Comment by thejameskyle** [comment]
http://goo.gl/forms/DtmQnUXNSE

**Comment by joshmanders** [comment]
Closing my PR and signing form. Thanks for this everyone.

**Comment by thejameskyle** [comment]
Google forms is being shitty, the app is not working

**Comment by joshmanders** [comment]
Hmm must have been a glitch, seems to be working for me.

**Comment by thejameskyle** [comment]
Reenabled form

**Comment by nicatronTg** [comment]
I suggest using Typeform -- they're good at handling load and it looks nice, design wise.

**Comment by thejameskyle** [comment]
Google Spreadsheets/forms is being shitty and I can't share the link to the responses

<img width="661" alt="screen shot 2016-01-14 at 4 51 46 pm" src="https://cloud.githubusercontent.com/assets/952783/12342428/1f7d36b0-badf-11e5-8ee1-7ff3c62f9a5e.png">

**Comment by thejameskyle** [comment]
Okay this should work

https://docs.google.com/spreadsheets/d/1oGsg02jS-PnlIMJ3OlWIOEmhtG-udTwuDz_vsQPBHKs/edit?usp=sharing

**Comment by thejameskyle** [comment]
Good point raised: How do we verify that these are the actual GitHub users?

**Comment by joshmanders** [comment]
Maybe restrict editing on that spreadsheet. It looks like someone is manually trying to add their name.

**Comment by thejameskyle** [comment]
Fuck– people suck. I can't lock it to view only because google keeps erroring out

**Comment by joshmanders** [comment]
An open letter and petition signing from OSS developers to Github, the first thing to kill Google.

**Comment by nafg** [comment]
How about track "signatures" with an issue, asking people to comment +1 on it.

It be a good way to bring the point home...

**Comment by NoahAndrews** [comment]
@thejameskyle has an really important point. The Google form has no verification. It's far better to do it via git, since I can look at the commit history and see each person adding their signature using their real account.

**Comment by domenic** [comment]
What is the threat model for "no verification"? Someone might sign this who doesn't use GitHub? Are we really worried about that? What horrible consequences would it have?

**Comment by joshmanders** [comment]
@domenic you're right, there are no REAL horrible consequences. Worst case scenario someone tags Bill Gates and Bill Gates reaches out to have it removed.

**Comment by NoahAndrews** [comment]
No, I see the threat model as someone impersonating someone else. What's to stop me from saying that I'm John the maintainer of OpenToaster? It's also about lending credibility to the signatures. With the signatures added via pull request, I can verify that x number of people *actually* signed the letter, not just that someone says they did.

**Comment by joshmanders** [comment]
@NoahAndrews there would have to be things put in place to make sure people don't create merge conflicts, it's not really easy for Kyle or anyone else to just go through and merge PR's when there's 60+ and everyone wants to put their names in different areas, or rearrange the names.

**Comment by NoahAndrews** [comment]
Ah right, forgot about the ordering issue. That is tricky.

**Comment by chuyskywalker** [comment]
Could just add a "signed" directory where people commit a `my.name.md` file with their name in it. Have a jekyll page pull those in as signatures? No merge conflicts, mostly automatic updates.

**Comment by nicatronTg** [comment]
Use CLAHub: https://www.clahub.com/

People can just authorize their Github account and "sign" the CLA that says they support the petition. I'm not sure if you can get a public list, but you can certainly copy and paste.

**Comment by runspired** [comment]
Or, to sign, you just have to +1 an open issue :trollface:

**Comment by Codeusa** [comment]
I think you should restore the initial signers you removed in the last pull, as to show immediately what big repo owners are throwing support, removing them takes away from the message as you have to click through to another page just to see a table of names.

**Comment by joshmanders** [comment]
@Codeusa I don't think the size of the projects should matter. We're a community as a whole whether it's someone with a project with 10 stars, or someone with a project of 10,000 stars.

**Comment by kangax** [comment]
The last drop was @mrdoob's tweet yesterday morning which resonated with how I often feel about issues. I replied to him and saw @dmethvin (of jQuery), @ariya (of PhantomJS), and @nzakas (of ESLint) immediately chime in with exactly same concerns. I realized that all of our projects combined (5K+, 10K+, 30K+, etc. stars) have enough weight between us to make a case that will probably be heard and listened to. So this was indeed original motivation (powered by other folks from Babel, React, Lodash, WHATWG, etc.). Now that the letter had some publicity and we have over 500 people signing and expressing same concerns, it probably doesn't matter. It's a bit of a delicate issue: on one hand we don't want to come off elitist (as @kittens mentions in https://github.com/dear-github/dear-github/pull/78), on the other hand, there's some point in showing that some of the most popular projects (and so @github's biggest users and communities) are behind it.

**Comment by jmm** [comment]
@kangax I agree with #78. Just listing some of the projects (maybe with star count) instead of individuals might be a reasonable compromise. Of course some people may resent that their project isn't mentioned though, so I don't know. Like you said, it no longer really seems necessary to prove that it's endorsed by maintainers of popular projects, as well as many smaller projects.

**Comment by kangax** [comment]
Yep, maybe we could mention top 5 biggest projects behind — jQuery w. 37K, three.js w. 22K, lodash & babel w. 13K, etc.

**Comment by runspired** [comment]
@kangax there's probably 2 dozen 10k star plus repos in there just glancing at the list (I'm with Hammer.js, we're 5 away from 13k).  It's hard to pick just a few and put them at the top.

**Comment by dmethvin** [comment]
I would be fine re-signing. Or to make our point, let's just create an issue here and have everyone on Github come in and +1 the thing.

**Comment by kangax** [comment]
Ideally, each signature in a list would have corresponding project and stars would be fetched automatically. That way it could be sorted by stars and just show better picture.

**Comment by jmm** [comment]
@kangax You mean in or derived from the spreadsheet? The ship has probably sailed on that unless someone wants to go through and normalize all the rows to have an `{{orgname}}{{reponame}}` column, for starters.

**Comment by kittens** [comment]
Has the goal of this letter not been accomplished? Why does anything else need to happen regarding signatures?

**Comment by thejameskyle** [comment]
I agree, everything that is going to happen with this letter has already happened. We'll see if GitHub chooses to respond.

**Comment by kangax** [comment]
The goal is not accomplished IMO, but don't think we need to change anything. I would want official github response at the very least (not just 1 comment by a "new community manager" on HN).

**Comment by nzakas** [comment]
Agree with @kangax. Having people continue to sign and support this letter will show GitHub that this isn't just a few people who share this opinion.

**Comment by thejameskyle** [comment]
I think people can continue signing it, I'm not going to shut the form down. I don't think we need to make any other changes to this repo though

**Comment by kittens** [comment]
Just want to clarify that I didn't mean shutdown the signature form, just that the signatures on the actual document are now pointless.

**Comment by lukejacksonn** [comment]
+1

**Comment by erkinalp** [comment]
+ erkinalp/strfry

**Comment by onebree** [comment]
I think this issue can be closed now...
